Dart/src/com/jetbrains/lang/dart/ide/errorTreeView/AnalysisServerStatusAction.java

```diff
@@ -28,12 +28,16 @@ public AnalysisServerStatusAction() {
   public void actionPerformed(AnActionEvent e) {
     DartFeedbackBuilder builder = DartFeedbackBuilder.getFeedbackBuilder();
 
-    if (MessageDialogBuilder.yesNo(builder.title(), builder.prompt())
-          .icon(Messages.getQuestionIcon())
-          .yesText(builder.label())
-          .show() == Messages.YES) {
-      builder.sendFeedback(e.getProject());
+    if (builder.isPromptRequired()) {
```

Comment:
This seems like a premature optimization of the UX to me. I think many people do not pay attention to the status line and if they have not seen the message then clicking this button is going to have a very surprising effect.

I'm moving the MessageDialogBuilder construction into the DartFeedbackBuilder.